content/docs/get-started/index.md

```diff
@@ -59,43 +52,40 @@ save(
 )
 ```
 
-Here, we loaded a well-known
-[Iris flower dataset](https://archive.ics.uci.edu/ml/datasets/iris) with
-`scikit-learn` and trained a simple classifier. Instead of pickling the model,
-we persisted it using MLEM's `save` API.
-
-`scikit-learn` is just an example of many supported ML frameworks. Check out the
-[full list here](/doc/object-reference/model).
-
-Now, let's see what MLEM saved by using the `tree` terminal command:
+Here, we loaded a well-known [Iris flower dataset] with scikit-learn and trained
+a simple classifier (see the full list of [supported ML frameworks]). Instead of
+pickling the model, we captured it using `mlem.api.save`, which created the
```

Comment:
"To persist" doesn't really mean [what we want it to](https://github.com/iterative/dvc.org/issues/2960). "Save" was better anyway (it's simple and it matches the MLEM fn name).